dashboard/app/helpers/restricted_markdown_helper.rb

```diff
@@ -0,0 +1,38 @@
+require 'redcarpet'
+require 'redcarpet/render_strip'
+
+# A markdown renderer that strips everything, except for codespans (via backticks)
+module RestrictedMarkdownHelper
```

Comment:
I like the specific name `CodeSpanOnly` for the renderer; why not maintain that throughout? `CodeSpanOnlyHelper` and `CodeSpanRenderer`?